docs/examples/overlaypanel/animationExample.js

```diff
@@ -0,0 +1,82 @@
+// @flow strict
+import { Fragment, type Node, useState, useCallback } from 'react';
+import {
+  Box,
+  Button,
+  CompositeZIndex,
+  FixedZIndex,
+  Flex,
+  IconButton,
+  Layer,
+  OverlayPanel,
+} from 'gestalt';
+
+export default function AnimationExample(): Node {
+  const [shouldShow, setShouldShow] = useState(false);
+  const HEADER_ZINDEX = new FixedZIndex(10);
```

Comment:
nit: define these outside of the function so they're not recreated on every render

Reply:
DONE